hw/top_earlgrey/data/ip/chip_otbn_testplan.hjson

```diff
@@ -0,0 +1,147 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  name: otbn
+  testpoints: [
+    {
+      name: chip_sw_otbn_isatest
+      descr: '''Run every OTBN instruction and check final state.
+
+                Run the existing OTBN smoke test which uses every instruction
+                (hw/ip/otbn/dv/smoke/smoke_test.s). Check the final register state matches the
+                expected values (in hw/ip/otbn/dv/smoke/smoke_expected.txt). Check OTBN completed
+                successfully without errors.
```

Comment:
I think we're going to have to tweak the smoke test so that it can check expected values as it runs. Maybe this should end up in an "extended_smoke_test.s"?

Reply:
I'd prefer Ibex to do the actual checking, though we will need to modify the smoke test to dump things, likely as it goes along rather than just at the end. I'll add some extra description for this.